lib/data_migrate/data_migrator_five.rb

```diff
@@ -1,10 +1,11 @@
 # frozen_string_literal: true
 
 require "active_record"
+require "data_migrate/config"
 
 module DataMigrate
   class DataMigrator < ActiveRecord::Migrator
-    self.migrations_paths = ["db/data"]
+    self.migrations_paths = [DataMigrate.config.data_migrations_path]
```

Comment:
@EnomaDebby is this consistent with the work you did for data_migrations path?

Reply:
It is

Reply:
Yes, it is